db/schema.rb

```diff
@@ -215,18 +215,19 @@
   add_index "faqs", ["category"], name: "index_faqs_on_category", using: :btree
 
   create_table "images", force: :cascade do |t|
-    t.string   "description",       limit: 255
-    t.integer  "album_id",          limit: 4
-    t.string   "foto_file_name",    limit: 255
-    t.string   "foto_content_type", limit: 255
-    t.integer  "foto_file_size",    limit: 4
-    t.datetime "foto_updated_at"
+    t.integer  "album_id",        limit: 4
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.datetime "captured"
-    t.integer  "subcategory_id",    limit: 4
+    t.string   "file",            limit: 255
+    t.string   "filename",        limit: 255
+    t.integer  "photographer_id", limit: 4
+    t.integer  "width",           limit: 4
+    t.integer  "height",          limit: 4
   end
 
+  add_index "images", ["file"], name: "index_images_on_file", using: :btree
+  add_index "images", ["filename"], name: "index_images_on_filename", using: :btree
```

Comment:
Probably needs more indices. You should add at least:

* photographer_id
* captured
* album_id